kit/ChildSession.cpp

```diff
@@ -1634,6 +1634,8 @@ bool ChildSession::insertFile(const StringVector& tokens)
 
     SigUtil::addActivity(getId(), "insertFile " + type);
 
+    LOG_TRC("InsertFile with arguments: " << type << ": " << (data.empty() ? name : std::string("binary data")));
```

Comment:
Why not put this at the top of the function?

Reply:
because I want to log params which we get in above lines